src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -11,10 +11,11 @@
 import mate.jdbc.exception.DataProcessingException;
 import mate.jdbc.lib.Dao;
 import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.ManufacturerService;
 import mate.jdbc.util.ConnectionUtil;
 
 @Dao
-public class ManufacturerDaoImpl implements ManufacturerDao {
+public class ManufacturerDaoImpl implements ManufacturerService {
```

Comment:
ManufacturerDaoImpl implements ManufacturerDao
You need to create layer Dao and layer Service.
You mix them. Why?